src/components/learner-credit-management/data/hooks/useOfferRedemptions.js

```diff
@@ -29,19 +30,27 @@ const applySortByToOptions = (sortBy, options) => {
   });
 };
 
-const applyFiltersToOptions = (filters, options) => {
+const applyFiltersToOptions = (filters, options, fetchTransactions = null) => {
   const courseProductLineSearchQuery = filters?.find(filter => filter.id === 'courseProductLine')?.value;
-  const searchQuery = filters?.find(filter => filter.id.toLowerCase() === 'enrollment details')?.value;
+  const searchQuery = filters?.find(filter => filter.id.toLowerCase() === 'enrollmentdetails')?.value;
```

Comment:
small bug with filter ID - toLowerCase doesn't seem to be adding a space to camel cased words

Reply:
Great catch! Thanks for resolving for both data sources 💯 

[context] Originally, the `id` was based on the `Header` ("Enrollment details") which is why the `toLowerCase()` is there. Since then, the `accessor` attribute was added to the column which changed the `id` to `enrollmentDetails`.

[suggestion/nit, non-blocking] at this point we could probably do without the `.toLowerCase()` and just rely on the actual accessor being used instead (i.e., `enrollmentDetails`), similar to `courseProductLine` above.